test/runtests.jl

```diff
@@ -1,14 +1,11 @@
 using BlockDiagonals
 using ChainRulesCore
 using ChainRulesTestUtils
-using Documenter
 using FiniteDifferences # For overloading to_vec
 using Test
 using LinearAlgebra
 
 @testset "BlockDiagonals" begin
-    # The doctests fail on x86, so only run them on 64-bit hardware
-    Sys.WORD_SIZE == 64 && doctest(BlockDiagonals)
```

Comment:
Why not just add a version check here?

Reply:
Could do.
it does mean keeping it up to date in two places (the CI.yml and here)
Do you think it's work it?
If so I will revert those change to that bit and add it.

Reply:
I don't really contribute to this package, so I think I'll leave it up to the maintainers as to what they want. For things, I work on regularly having doctests run by default locally is worth it for me. I tend to forget about running documenter otherwise and we aren't switching Julia versions every month. Others may feel differently here.

Reply:
Alternative PR
https://github.com/invenia/BlockDiagonals.jl/pull/65